dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/RefinementSolver.java

```diff
@@ -301,6 +306,54 @@ private void runInternal(SolverContext ctx, ProverEnvironment prover, Verificati
         logger.info("Verification finished with result " + res);
     }
 
+    private void analyzeInconclusiveness(VerificationTask task, RefinementTrace propertyTrace, Context analysisContext) {
+        final AliasAnalysis alias = analysisContext.get(AliasAnalysis.class);
+        if (alias == null) {
+            return;
+        }
+        SyntacticContextAnalysis synContext = analysisContext.get(SyntacticContextAnalysis.class);
+        if (synContext == null) {
+            synContext = newInstance(task.getProgram());
+        }
+
+        final DNF<CoreLiteral> lastReasons = propertyTrace.getFinalIteration().inconsistencyReasons;
+        for (Conjunction<CoreLiteral> reason : lastReasons.getCubes()) {
+            for (CoreLiteral literal : reason.getLiterals()) {
+                if (literal instanceof RelLiteral relLit && doesViolateAliasingRules(relLit, alias)) {
+                    final Event e1 = relLit.getSource();
+                    final Event e2 = relLit.getTarget();
+
+                    final StringBuilder builder = new StringBuilder();
+                    builder.append("Found aliasing problem between:\n");
```

Comment:
Maybe you can specify that the solver found an execution where both alias but the alias analysis says this is impossible.

Reply:
I can update the error to `Found unexpected aliasing between: `, which I think is perfectly understandable to us.